src/main/java/duke/utils/AException.java

```diff
@@ -0,0 +1,66 @@
+package duke.utils;
```

Comment:
should be a subclass of exception. 
for multiple exception, should subclass DukeException class
Rename the class to more meaningful names such as DukeException